schema.graphql

```diff
@@ -229,57 +263,113 @@ input AddOpticalImageInput {
 }
 
 type Query {
-  dataset(id: String!): Dataset
-  datasetByName(name: String!): Dataset
+  # If JWT is missing, only public information is returned;
+  # otherwise, user access rights are taken into account;
+  # For consistency, every query has 'jwt' parameter.
+
+  dataset(jwt: String, id: String!): Dataset
+  user(jwt: String, id: String!): UserProfile
+  group(jwt: String, id: String!): Group
+  project(jwt: String, id: String!): Project
```

Comment:
@lomereiter For consistency, should we pass jwt as the last argument here as well?

Reply:
Feel free to move it around, but in principle it doesn't matter as all arguments are named in GraphQL.

Reply:
@intsco shall we perhaps get rid of JWT as a parameter and just use `Authorization: Bearer ...` header? GraphiQL doesn't support setting custom headers, but e.g. https://github.com/andev-software/graphql-ide does, and it's even more convenient than copy&pasting the token all the time.